src/app/static/src/app/store/modelCalibrate/actions.ts

```diff
@@ -18,10 +18,20 @@ import {CalibrateResultWithType, Dict, ModelOutputTabs} from "../../types";
 import {DownloadResultsMutation} from "../downloadResults/mutations";
 import {PlottingSelectionsMutations} from "../plottingSelections/mutations";
 import { ModelOutputMutation } from "../modelOutput/mutations";
+import { BarchartSelections, BubblePlotSelections, ChoroplethSelections, TableSelections } from "../plottingSelections/plottingSelections";
 
-type ResultDataPayload = {
-    indicatorId: string,
-    tab: ModelOutputTabs
+export type ResultDataPayload = {
```

Comment:
Possibly slightly clearer, as `ResultDataPayload` sounds like it's going to be the type of the result data itself

```suggestion
export type GetResultDataPayload = {
```